### DISCUSSION TITLE
Elevate current shell

### DISCUSSION THREAD

**Discussion by Jack-Chronicle** [microsoft/sudo#99] [Ideas]
Would really like to see the ability to elevate the current terminal shell with a command, rather than having to open an elevated prompt instead. I prefer opening non-elevated ones, but sometimes I end up needing to run elevated commands and would rather not  have to open a new shell to do so (even if the command isn't too long, it can clutter the screen with more terminals)

And having individual tabs elevated in Windows Terminal would let us have one window to run commands at different levels without needing to constantly confirm the UAC for each and every command, or have to add sudo to each command that needs to be elevated

**Comment by zadjii-msft** [comment]
This is totally possible today if you put sudo into `inline` mode. That's how I usually run it - does that not work for you/?

**Reply by Jack-Chronicle** [reply]
That does seem to work now, it may not have when I made the initial post. Haven't been keeping up with it, so not sure when the update occurred. But thanks for letting me know it's been changed
